Automattic/vip-go-mu-plugins
# Liveblog: Purge Varnish for each new liveblog entry.

Given that Varnish caches posts for 30 minutes, a liveblog post can contain important meta data that is as much as 30 minutes out of date. This is because the post includes data about the last updated timestamp and this is included in the cached output.

This data can therefore differ from the actual entries, which are loaded asynchronously.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Sandbox a test site
1. Apply the patch
1. Start a new Liveblog, and load it in the browser, in incognito
1. Reload until you see the `X-Cache: hit` header
1. Note the associated meta data in the source code, it should include `latest_entry_timestamp`
1. In a logged-in browser, add a new entry
1. The new entry should appear in the incognito window
1. Reload the page
1. The liveblog entries should remain the same
1. Check the meta data again - the timestamp should have changed, and the `X-Cache: miss` header should be present
